app/services/inbound_webhooks/process_service.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module InboundWebhooks
+  class ProcessService < BaseService
+    WEBHOOK_HANDLER_SERVICES = {
+      stripe: PaymentProviders::Stripe::HandleIncomingWebhookService
+    }
+
+    def initialize(inbound_webhook:)
+      @inbound_webhook = inbound_webhook
+
+      super
+    end
+
+    def call
+      inbound_webhook.processing!
```

Comment:
definitely need some precondition checking here I think, what if the web hook is already in processing/ processed?

Reply:
yes, thanks for spotting... I am considering return if the status is processed or failed and let pending and processing continue.

The reason behind letting processing continue is if a sidekiq worker is killed while processing the webhook it will be in processing status...

what do you think?